addon/components/besluit-plugin-card.hbs

```diff
@@ -15,6 +15,20 @@
       <AuButton {{on "click" this.insertArticle}} @icon="plus-text" @iconAlignment="left" @skin="link-secondary">
         Voeg artikel in
       </AuButton>
+      {{#if this.articleElement}}
+        <br />
+        <AuButton {{on 'click' this.deleteArticle}} @icon="plus-text" @iconAlignment="left" @skin="link-secondary">
```

Comment:
These buttons should dissapear when they are not relevant i.e. no article to remove, or article is already at the top

Reply:
The button should disappear if you have no article anymore, that's a bug, good catch, I will fix those, checking if the article is already at the top was too much querying so I think is not worth in a perspective of performance vs UX. It can be improved if you think is worth, take into account that this would have to run on every selection change and not alone, along with other plugins

Reply:
this will probably be easier after https://github.com/lblod/ember-rdfa-editor/pull/211

Reply:
cause there you get nodes in document order so you can determine order